app/src/main/java/com/github/se/cyrcle/io/MapTileStorage.kt

```diff
@@ -0,0 +1,46 @@
+package com.github.se.cyrcle.io
+
+import java.io.File
+import java.io.IOException
+
+private const val CACHE_DIR = "./cache/map"
```

Comment:
did you consider using android's cache directory ?

Reply:
Yes, but no, as the goal of this is to not be a cache for the application, but rather some content that should be store directly (think of downloading a part of the worlds). But what your comment makes me realize is that the directory name isn't well define, as this is no cache. This will be address

Reply:
Ok I understand.